ignite/templates/app/stargate/app/app.go.plush

```diff
@@ -687,6 +702,7 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	paramsKeeper := paramskeeper.NewKeeper(appCodec, legacyAmino, key, tkey)
 
 	paramsKeeper.Subspace(authtypes.ModuleName)
+	paramsKeeper.Subspace(authz.ModuleName)
```

Comment:
```suggestion
```

Reply:
@ilgooz why don't we need this here?

Reply:
in gaiad is in this way O_o

Reply:
@RaulBernal can you please post the link to the related code line in Gaia?